src/validation.cpp

```diff
@@ -1003,10 +1003,8 @@ bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransa
  * Return transaction in txOut, and if it was found inside a block, its hash is placed in hashBlock.
  * If blockIndex is provided, the transaction is fetched from the corresponding block.
  */
-bool GetTransaction(const uint256& hash, CTransactionRef& txOut, const Consensus::Params& consensusParams, uint256& hashBlock, bool fAllowSlow, CBlockIndex* blockIndex)
+bool GetTransaction(const uint256& hash, CTransactionRef& txOut, const Consensus::Params& consensusParams, uint256& hashBlock, CBlockIndex* blockIndex)
 {
-    CBlockIndex* pindexSlow = blockIndex;
```

Comment:
After this removal, where do you still use `blockIndex`?  It seems to me that "lookup by block index" is now also removed.  Is that intentional?

Reply:
Oops, no, fixing.